core/unit_test/CMakeLists.txt

```diff
@@ -52,18 +59,25 @@ endforeach()
 if(${Cabana_ENABLE_MPI})
   foreach(_device ${CABANA_SUPPORTED_DEVICES})
     if(Cabana_ENABLE_${_device})
+      set(_dir ${CMAKE_CURRENT_BINARY_DIR}/${_device})
+      file(MAKE_DIRECTORY ${_dir})
```

Comment:
What happens if this directory has already been created? Does this fail or throw a warning?

Reply:
Neither.

Reply:
CMake works like `mkdir -p`:
```
$ cmake -E make_directory 1; echo $?
0
$ cmake -E make_directory 1; echo $?
0
```

Reply:
got it